submissions/LGeorge007/DOM-practice/script.js

```diff
@@ -0,0 +1,175 @@
+        titleFeatures: "Особенности",
+        textFeatures: "Большое существо – занимает две клетки на поле боя. Летающее существо. Воскрешение союзников – юниты способны один раз за бой воскресить определённое число погибших дружественных юнитов в целевом отряде из расчёта 100 ед. здоровья воскрешённых на одного архангела. Воодушевление => +1 ед. к боевому духу всех союзных воинов; действует даже после уничтожения архангелов. Ненависть к дьяволам и архидьяволам => +50% дополнительного урона при атаке.",
+        titleTactics: "Тактика",
+        textTactics: "Архангелы являются одними из сильнейших юнитов в игре. Они обладают самыми высокими тактическими показателями среди всех остальных замковых юнитов своего уровня, незначительно уступая лишь некоторым из них в здоровье, скорости и уроне. При этом, однако, весьма серьёзным недостатком архангелов, помимо высокой стоимости, является отсутствие какого-либо магического иммунитета. Это означает, что архангелы не только будут получать полный урон от наступательной магии вражеских героев-магов, но и могут быть легко выведены из боя с помощью, скажем, такого заклинания как \"Ослепление\". Тактика игры за архангелов во многом идентична тактике игры за обычных ангелов, хотя способность \"Воскрешение\" может очень серьёзно повлиять на ход битвы. Правда, использовать её следует грамотно, чтобы не воскресить меньшее число юнитов, чем было бы возможно. Также следует помнить, что архангелы, само собой, не могут воскрешать нежить, элементалей и механизмы – следовательно, в армии, где в большом количестве присутствуют данные юниты, архангелы будут не так полезны. Наконец, если отряд архангелов достаточно велик, будет разумно разделить его на две или же более части – хотя сила воскрешения каждого отряда упадёт, архангелы смогут восполнить потери сразу в нескольких союзных отрядах. Впрочем, необходимость в таком разделении отпадает, если архангелы находятся в армии героя, специализирующегося на магии Воды и знающего такое заклинание как \"Фантом\", ведь фантомы архангелов воскрешают ничуть не хуже, чем сами архангелы.",
+        titleStrenghts: "Сильные стороны",
+        textStrenghts: "Среди улучшенных юнитов 7-го уровня: Самый высокий показатель атаки. Самый высокий показатель защиты. Второй по величине показатель скорости (выше лишь у фениксов). Второй по величине показатель урона (выше лишь у титанов). Отсутствие разброса урона. Воскрешение союзников. Как следствие: Самая высокая стоимость. Крайне дорогая отстройка Улучшенного Портала Славы .",
+        titleWeaknesses: "Слабые стороны",
+        textWeaknesses: "Отсутствие магического иммунитета с учётом стоимости найма и среднего здоровья.",
+        img: "img/archangel/archangel.jpg"
+    },
+
+];
+
+const render = function (item) {
+
+        content.innerHTML = "";
+        const unit = listOfUnits.filter(unit => unit.titleMenuEng == item)[0];
+        let templateCard = `
+            <h1>${unit.titleDescription}</h1>
```

Comment:
This line can be created with titleRus and titleEng inside code. In this case, you change unit name in one place
```suggestion
     <h1>"${titleRus} (ориг. ;${titleEng})"<h1>
```